File: SoloLearn Codes/2D Map/TwoDMap.cs
using System;
using System.Collections.Generic;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] ps = Console.ReadLine().Trim().Split(',');
            List<(int, int)> M = new List<(int, int)>();
            for (int i = 0; i < ps.Length; i++)
                for (int j = 0; j < ps[i].Length; j++) if (ps[i][j] == 'P') M.Add((i, j));
            Console.WriteLine(Math.Abs(M[0].Item1 - M[1].Item1) + Math.Abs(M[0].Item2 - M[1].Item2));
        }
    }
}

File: SoloLearn Codes/2D Map/twod_map.swift
let m = readLine()!.split(separator: ",")
var p = [(Int, Int)]()
for (i, r) in m.enumerated() {
    for (j, c) in r.enumerated() {
        if c == "P" {
            p.append((i, j))
        }
    }
}
print(abs(p[0].1 - p[1].1) + abs(p[0].0 - p[1].0))